app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
-            if self.email == current_app.config['ADMIN_EMAIL']:
+            if not User.query.first():
+                Role.insert_roles()
                 self.role = Role.query.filter_by(
                     permissions=Permission.ADMINISTER).first()
-            if self.role is None:
+            else:
```

Comment:
Only reason I can think is if it was serving as a redundancy. i.e. in the previous implementation, if there is no ADMINISTER role, self.role will still be None after the previous block, and the statement'll assign the user the default role. With your proposed change, there's no way to catch that. However, don't think that matters--since we can just ensure there is an ADMINISTER role--thus they should both behave identically, but yours looks cleaner.